src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+
+class HelloWorld {
+    private String string;
```

Comment:
it isn't very informative variable name

Reply:
Ok. I'll try be more accurate in future. Thank you.